onadata/libs/serializers/organization_serializer.py

```diff
@@ -32,6 +32,7 @@ class OrganizationSerializer(serializers.HyperlinkedModelSerializer):
     creator = serializers.HyperlinkedRelatedField(
         view_name='user-detail', lookup_field='username', read_only=True)
     users = serializers.SerializerMethodField()
+    is_org = serializers.SerializerMethodField()
```

Comment:
What's the need for the `is_org` field? Since this is the Organization serializer... I'd expect this to only be used for organizations....? Is that not the case?